src/Lean/Elab/Coinductive.lean

```diff
@@ -0,0 +1,217 @@
+/-
+Copyright (c) 2020 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: William Sørensen
+-/
+prelude
+import Lean.Elab.Command
+import Lean.Elab.DeclarationRange
+import Lean.Elab.Exception
+import Lean.Elab.Inductive
+import Lean.Util.Trace
+
+namespace Lean.Elab.Command
+
+open Lean Lean.Elab Lean.Elab.Term Lean.Parser Command
+
+builtin_initialize
+  registerTraceClass `Elab.CoInductive
+
+structure CoInductiveView.CtorView where
```

Comment:
```suggestion
structure CoinductiveView.CtorView where
```

Very minor nit, but I'd consider coinductive to be one word, and thus not capitalize the `I` in names